### PR TITLE
chore: Fix typo in the name of an internal bazel target

### PR DESCRIPTION
**Pull request by aterga**
This PR fixes a typo in the name of an internal bazel target.

### CODE REVIEW

File: publish/binaries/BUILD.bazel [hunk 1]
```diff
@@ -31,6 +31,7 @@ ALL_BINARIES = {
     "orchestrator": "//rs/orchestrator",
     "sandbox_launcher": "//rs/canister_sandbox:sandbox_launcher",
     "sns": "//rs/sns/cli:sns",
+    "sync-with-released-nervous-system-wasms": "//rs/nervous_system/tools/sync-with-released-nervous-system-wasms",
```
Comment (basvandijk) [inline publish/binaries/BUILD.bazel:34]: Why does this tool need to be published? Wouldn't we just use this internally in the ic repo where we can just `bazel run //rs/nervous_system/tools/sync-with-released-nervous-system-wasms ...` it?

Comment (aterga) [reply publish/binaries/BUILD.bazel:34]: We want this action to be done via a bot that would then create a PR in this repo. 

I was thinking that the bot could be added to https://github.com/dfinity-ops/release

Is there a better way?

Comment (basvandijk) [reply publish/binaries/BUILD.bazel:34]: Why not via a GitHub action in this repo similar to what @NikolaMilosa is doing for `mainnet_revisions.json` in https://github.com/dfinity/ic/pull/3085?

Comment (aterga) [reply publish/binaries/BUILD.bazel:34]: Agreed. But adding it to that repo would require publishing the tool from this repo, I think. 

So, may I get an approval, @basvandijk?

Comment (basvandijk) [reply publish/binaries/BUILD.bazel:34]: WDYM with "that repo". What Nikola is doing is adding a GH action to "this repo".

Comment (aterga) [reply publish/binaries/BUILD.bazel:34]: Ah, so I misunderstood. 

Yes, keeping all this within the same repo seems better. 

(I wonder how we can build / run a bazel target from an action. But let's move that discussion outside this PR.)

So I'm removing the publishing of this target from this PR.

Thanks!

Comment (aterga) [reply publish/binaries/BUILD.bazel:34]:
```suggestion
```

File: .github/repo_policies/BOT_APPROVED_FILES [hunk 2]
```diff
@@ -8,3 +8,4 @@ ic-os/hostos/context/docker-base.prod
 ic-os/setupos/context/docker-base.dev
 ic-os/setupos/context/docker-base.prod
 testnet/mainnet_revisions.json
+mainnet-canisters.json
```
[the remark this answers is not in the document]
Comment (aterga) [reply .github/repo_policies/BOT_APPROVED_FILES:11]:
```suggestion
```

File: publish/binaries/BUILD.bazel (hunk 1 above)
Comment (basvandijk) [reply publish/binaries/BUILD.bazel]: >I wonder how we can build / run a bazel target from an action. But let's move that discussion outside this PR.

```
bazel run --config=ci //rs/nervous_system/tools/sync-with-released-nervous-system-wasms ...
```

File: .github/repo_policies/BOT_APPROVED_FILES (hunk 2 above)
[the remark this answers is not in the document]
Comment (basvandijk) [reply .github/repo_policies/BOT_APPROVED_FILES]: Well, I guess you could already add it. But you could also do it in the PR where you'll add an action that does the `bazel run --config=ci //rs/nervous_system/tools/sync-with-released-nervous-system-wasms ...`.

**Review by basvandijk — APPROVED** [review]

**Review by daniel-wong-dfinity-org — APPROVED** [review]

**Pull request merged**